src/components/table-ajax/table-ajax.stories.mdx

```diff
@@ -0,0 +1,208 @@
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import { PropsTable } from "@storybook/components";
+
+import countriesList from "../../../.storybook/utils/xhr/data/countries";
+import { enableMock } from "../../../.storybook/utils/xhr/xhr-mock";
+import { Table } from "../table";
+import { TableAjax, TableRow, TableCell, TableHeader } from ".";
+
+<Meta title="Table Ajax" parameters={{ info: { disable: true } }} />
+
+# Table Ajax
+
+- This component is the same as [Table](/components/table "Table"), but uses Ajax.
+- Ajax loads data from a specified source as needed, rather than data in the page markup.
+
+## Quick Start
+
+```jsx
+import {
+  TableAjax,
+  TableRow,
+  TableCell,
+  TableHeader,
+} from "carbon-react/lib/components/table-ajax";
+
```

Comment:
I think it was mentioned before that we only need to provide the import paths here. No need for the additional code below as the consumer can click `show code` on the examples.